packages/sanity/src/_internal/cli/actions/deploy/undeployAction.ts

```diff
@@ -37,16 +43,25 @@ export default async function undeployStudioAction(
   output.print('')
 
   const url = `https://${chalk.yellow(userApplication.appHost)}.sanity.studio`
-  const shouldUndeploy = await prompt.single({
-    type: 'confirm',
-    default: false,
-    message: `This will undeploy ${url} and make it unavailable for your users.
+
+  /**
+   * Unattended mode means that if there are any prompts it will use `YES` for them but will no change anything that doesn't have a prompt
+   */
+  const unattendedMode = Boolean(flags.yes || flags.y)
+
+  // If it is in unattended mode, we don't want to prompt
```

Comment:
Q: is this comment incorrect, should it be `If it is NOT in unattended mode...`, or have I misunderstood the logic?

Reply:
I think it's correct. I copied the wording from here:
https://github.com/sanity-io/sanity/blob/ddb65fa21a42856926331635e63b17ac420d529a/packages/sanity/src/_internal/cli/actions/build/buildAction.ts#L76C1-L77C1